scripts/promptflow-ci/requirement.txt

```diff
@@ -2,6 +2,7 @@ azure-storage-blob
 azure-identity
 markdown
 bs4
+numpy<2
```

Comment:
azureml-core doesn't work with numpy 2.0
Error message - `np.unicode_` was removed in the NumPy 2.0 release. Use `np.str_` instead.`